.github/workflows/main.yml

```
@@ -0,0 +1,70 @@
+name: ci
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: npm install
+    - run: npx aegir lint
+    - run: npx aegir dep-check -- -i wrtc -i electron-webrtc
```

```suggestion
    - run: npx aegir dep-check
```

No need to ignore wrtc and electron-webrtc